docs/gcp/osd_preexisting_vpc.md

```diff
@@ -0,0 +1,168 @@
+# Creating a OSD in GCP with Existing VPCs
+
+**Roberto Carratalá, Andrea Bozzoni**
+
+*Last updated 07/06/2022*
+
+> **Tip** The official documentation for installing a OSD cluster in GCP can be found [here](https://docs.openshift.com/dedicated/osd_cluster_create/creating-a-gcp-cluster.html).
+
+For deploy an OSD cluster in GCP using existing Virtual Private Cloud (VPC) you need to implement some prerequisites that you must create before starting the OpenShift Dedicated installation though the OCM:
+
+## Prerequisites
+
+* [gcloud CLI](https://cloud.google.com/sdk/gcloud)
+* [jq](https://stedolan.github.io/jq/download/)
+
+NOTE: Also the GCloud Shell can be used, and have the gcloud cli among other tools preinstalled.
```

Comment:
mention the two options CLI vs TF up here so that a person knows to skip straight to the TF section.

Reply:
ack